src/maintenance.js

```diff
@@ -86,6 +86,10 @@ module.exports.delete = async function({Namespace, rabbitManager, cfg, namespace
   assert(namespace.length <= 64 && /^[A-Za-z0-9_-]+$/.test(namespace));
   let owned = new RegExp(cfg.app.userConfigPermission.replace(/{{namespace}}/g, namespace));
 
+  let connections = await rabbitManager.connections(cfg.app.virtualhost);
+  await Promise.all(connections.map(
+    conn => rabbitManager.terminateConnection(conn.name, 'Namespace deleted')));
```

Comment:
Don't you want to filter on something?

Also I don't know what this method is supposed to do at all... deleting what? a namespace?

Why do we support that?

I feel like there is some coupling here that is wrong... in the concept...
The `Namespace` table is just to track which users exist... We should just remove that entry and allow exchanges/queues to exist when it expires...

Deleting the queues/exchanges only happens when queues have been unused for a very long time... and exchanges are only deleted if they are manually deleted... I don't mind having a way to do it, but why is it coupled with credentials in anyway?